src/Module/Contact.php

```diff
@@ -840,6 +846,7 @@ public static function content(array $parameters = [], $update = 0)
 			'$submit'     => DI::l10n()->t('Find'),
 			'$cmd'        => DI::args()->getCommand(),
 			'$contacts'   => $contacts,
```

Comment:
Shouldn't all value be indented to the same level?

Reply:
It's debatable when the key length varies a lot, like in this case. Evidently a choice had been made earlier to align short keys together and leave longer keys alone and I chose to respect it.